bridge/src/lib.rs

```diff
@@ -48,10 +48,10 @@ class!(TemporalBridge);
 
 methods!(
     TemporalBridge,
-    rtself,
+    _rtself, // somehow compiler is sure this is unused and insists on the "_"
```

Comment:
Weird. Could be a clippy bug. This macro is probably doing a lot of weird stuff.

I mentioned this before, and it's really worth taking the minute or two to do -- I would take all of the logic out of this macro and have the macro methods delegate to those real implementations so there's just the bare minimum inside the macro. It will help with IDE stuff too.

Reply:
Yep, got it on my list, will make sure to prioritise it — #26